api/resources_portal/test/dev_data/auth.json

```diff
@@ -0,0 +1,38 @@
+{
+  "auth_group": {
+    "0": {
```

Comment:
What's the purpose of the key `"0"`? Seems to be a prevalent pattern.

Reply:
When there are multiple rows needed for the table, I indicated them as keys 0 through the num rows-1. Does it make more sense to have them one-indexed, or to use some other format?

Reply:
I think a list structure would make more sense. Like:
```
{
  "auth_group": [
    {<row_1>},
    {<row_2},
  ],
  auth_group_permissions": [
  {<row_1>}
  ]
etc
```